packages/api-explorer/src/JsonForm/anyOf-custom-editor.js

```diff
@@ -0,0 +1,168 @@
+    this.switcher.replaceWith(switcherDiv)
+    this.addErrorMessageHtmlNode()
+    switcher.dispatchEvent(new Event('change'))
+    this.showErrorMessage('start-config')
+    setSwitcher(switcher)
+    return response
+  },
+  buildSwitcher() {
+    const self = this
+    const switcher = this.theme.getSwitcher(this.display_text)
+    switcher.multiple = true
+    switcher.selectedIndex = '-1'
+    
+    const switcherDiv = document.createElement('div')
+    const switcherLabel = document.createElement('label')
+    switcherLabel.style.fontWeight = 'bold'
+    switcherLabel.innerText = 'Select the schemas:'
+    switcherDiv.appendChild(switcherLabel)
+    switcherDiv.appendChild(switcher)
+    const choices = new Choices(switcher, {
+      placeholder: true,
+      placeholderValue: 'Select the schema',
+      removeItemButton: true,
+      duplicateItemsAllowed: false,
+      classNames: {
+        listItems: 'multiple-select-choices-list choices__list--multiple',
+        item: 'multiple-select-item choices__item'
+      }
+    })
+
+    switcher.addEventListener('change', e => {
+      e.preventDefault();
+      e.stopPropagation();
+      const selectedValues = Array.from(e.currentTarget.selectedOptions).map(selectedValue => selectedValue.value)
+      if (selectedValues.length === 0) {
+        self.showErrorMessage('empty-selection')
+      } else if (selectedValues.length === 1) {
+        self.hideErrorMessage()
+        self.switchEditor(self.display_text.indexOf(selectedValues[0]))
+      } else {
+        self.updateEditor(selectedValues)
+      }
+      self.onChange(true)
+      choices.hideDropdown()
+    });
+
+    return { switcher, switcherDiv }
+  },
+  updateEditor(selectedValues) {
+    const joinedValues = selectedValues.join('_')
+    const mergedEditorIndex = editorIndexesMap.get(joinedValues)
+    if (mergedEditorIndex) {
+      this.hideErrorMessage()
+      this.switchEditor(mergedEditorIndex);
+      return
+    }
+    const schemasIndexes = selectedValues.map(value => this.display_text.indexOf(value))
+    const schemas = this.types.filter((_, index) => schemasIndexes.includes(index))
+    const mergedSchemas = mergeSchemas(schemas, self)
+    if (!mergedSchemas) {
+      this.showErrorMessage('not-compatible')
+      return
+    }
+    this.hideErrorMessage()
+    const index = this.types.length
+    this.insertNewEditor(index, joinedValues, mergedSchemas)
+    this.switchEditor(index)
+  },
+  addErrorMessageHtmlNode() {
+    this.errorMessageHtmlNode = document.createElement('p')
+    this.errorMessageHtmlNode.style.color = 'red'
+    this.errorMessageHtmlNode.style.fontWeight = 'bold'
+    this.errorMessageHtmlNode.style.display = 'none'
+    this.errorMessageHtmlNode.style.marginTop = '10px'
+    this.container.appendChild(this.errorMessageHtmlNode)
+  },
+  showErrorMessage(type) {
+    const  errorsByType = {
+      'not-compatible': 'The selected schemas are not compatible!',
```

Comment:
Those should not be formatted by browser language?

Reply:
done